webapp/channels/src/sass/components/_modal.scss

```diff
@@ -762,7 +758,7 @@
         flex: 20;
     }
 
-    >div {
+    >div:not([data-simplebar]) {
```

Comment:
More implementation details of the new scroll bars bleeding into other parts of the code.